.github/workflows/cygwin_build.yml

```diff
@@ -17,11 +17,16 @@ jobs:
         env:
           HOME: ${{ runner.workspace }}/ardupilot
         run: |
-          choco install cygwin --params "/InstallDir:C:\Cygwin /NoStartMenu /NoAdmin"
-          choco install cygwin32-gcc-g++ python37 python37-future python37-lxml python37-pip python37-setuptools python37-wheel git libexpat procps gettext --source cygwin
+          choco install cygwin --verbose --params "/InstallDir:C:\Cygwin /NoStartMenu /NoAdmin"
+          type C:\ProgramData\chocolatey\logs\chocolatey.log
+          echo "c:\cygwin\bin" >> $GITHUB_PATH
+	  echo "$PATH"
```

Comment:
looks  like  there  is  some  space/tabs issue  there

Reply:
Thanks - fixed.

That's the third time on this PR :-/